pkg/specgen/generate/oci.go

```diff
@@ -18,6 +18,18 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func setProcfsOpts(s *specgen.SpecGenerator, g *generate.Generator) {
+	if s.ProcfsOpts == nil {
+		return
+	}
+	for i := range g.Config.Mounts {
+		if g.Config.Mounts[i].Destination == "/proc" {
+			g.Config.Mounts[i].Options = s.ProcfsOpts
```

Comment:
Should we append here? Or try and merge them?

Reply:
I think it is better to replace them, how would we deal with something like "nodev", and then "dev"?